Tests/test_imagefont_bitmap.py

```diff
@@ -39,4 +40,22 @@ def test_similar():
         fill=(0, 0, 0),
         font=font_outline,
     )
-    assert_image_similar(im_bitmap, im_outline, 20)
+    assert_image_similar(im_bitmap, im_outline, 4)
+
+
+@skip_unless_feature("freetype2")
+@pytest.mark.parametrize("bpp", (1, 2, 4, 8))
+def test_bitmap_font(bpp):
```

Comment:
This test is sufficient to replace the test above. Would it make sense to remove `test_imagefont_bitmap.test_similar` and move this new test to `test_imagefont`?

Reply:
Okay, let's do that.

Reply:
Done, also added Raqm version of test.